plugin.json

```diff
@@ -1,12 +1,17 @@
 {
     "id": "pi-system",
-    "version": "1.3.1",
+    "version": "1.3.2",
     "meta": {
         "label": "PI System",
         "description": "Retrieve data from your OSIsoft PI System servers",
         "author": "Dataiku (Alex Bourret)",
         "icon": "icon-pi-system icon-cogs",
-        "tags": ["Connector"],
+        "category": "Connect",
```

Comment:
❓ do we really want to remove "Connector" from tags? 
Shouldn't the category be "connectors" or "connector" ? Though I can't seem to find category being used elsewhere much. 
What is driving this change anyway - it's not related to the bug fix.